### PR TITLE
Add scrollspy behavior to docs

### PR DESCRIPTION
**Pull request by jquense**
fun little thing

### CODE REVIEW

**Comment by taion** [comment]
Wow.

**Comment by jquense** [comment]
figured I'd make sure the advice we were giving folks about using react-waypoint actually made sense :P...and it did

**Comment by taion** [comment]
That was more of a "go away" than an actual recommendation, BTW.

**Comment by taion** [comment]
I mean, the recommendation to use the waypoint thing :p

**Comment by jquense** [comment]
hehe I know ;) 

the real reason is that I wanted to do something mindless and simple

File: docs/src/ComponentsPage.js [hunk 1]
```diff
@@ -210,62 +321,62 @@ const ComponentsPage = React.createClass({
...
-                        <NavItem href="#tabs">Tabs</NavItem>
-                        <NavItem href="#pagination">Pagination</NavItem>
-                        <NavItem href="#pager">Pager</NavItem>
+                      <SubNav href={sections.navigation} text="Navigation">
+                        <NavItem href={sections.navs}>Navs</NavItem>
+                        <NavItem href={sections.navbars}>Navbars</NavItem>
+                        <NavItem href={sections.crumbs}>Breadcrumbs</NavItem>
+                        <NavItem href={sections.tabs}>Tabs</NavItem>
+                        <NavItem href={sections.pagination}>Pagination</NavItem>
+                        <NavItem href={sections.overlays}>Pager</NavItem>
```
Comment (taion) [inline docs/src/ComponentsPage.js]: Should be `sections.pager`.

**Comment by taion** [comment]
LGTM other than that 1 comment

**Comment by AlexKVal** [comment]
Wow.. Nice :+1:

**Pull request merged**